### DISCUSSION TITLE
GitHub Pages don't show ReactApp

### DISCUSSION THREAD

**Discussion by mferdev** [community/community#40544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm trying to upload ReactApp to GithubPages.

When i deploy the web, the github page don't show anythink
#

**Comment by jfullstackdev** [comment]
seems that's working by now 

![image](https://user-images.githubusercontent.com/47092464/205065718-68c4c9f9-7cb5-4518-98d6-cda89aacf8d3.png)

**Reply by mferdev** [reply]
that's the dns, its works

thanks!
